packages/ia-topnav/src/primary-nav.js

```diff
@@ -134,14 +141,20 @@ class PrimaryNav extends TrackedElement {
     const mediaMenuTabIndex = this.openMenu === 'media' ? '' : '-1';
     return html`
       <nav>
-        <a class="link-home" href=${formatUrl('/', this.baseHost)} @click=${this.trackClick} data-event-click-tracking="${this.config.eventCategory}|NavHome">${icons.iaLogo}${logoWordmark}</a>
+        <a
+          class="link-home"
+          href=${formatUrl('/', this.baseHost)}
+          @click=${this.trackClick}
+          data-event-click-tracking="${this.config.eventCategory}|NavHome"
```

Comment:
q: do we need a title attribute since the link's text value doesn't explicitly say to go home?
```suggestion
          data-event-click-tracking="${this.config.eventCategory}|NavHome"
          title="Go home"
```